Cargo.toml

```diff
@@ -26,7 +26,7 @@ regex = "1.3.1"
 lazy_static = "1.4.0"
 
 [target.'cfg(target_os="linux")'.dependencies]
-procfs = "0.5.3"
+procfs = "0.7"
```

Comment:
Do you think it would be possible to rely on a fixed version? (right now it's 0.7.4, I think?)

Reply:
Sure!  But to be honest, I don't think it matters.  My reading of the [cargo docs](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html) suggest that a both `0.7` and `0.7.4` are both "caret requirements" and in both cases, cargo would allow upgrades to the latest `0.7` version.

Regardless, I am happy to make this change!